lib/generators/paperclip/paperclip_generator.rb

```diff
@@ -27,4 +29,10 @@ def migration_file_name
   def migration_class_name
     migration_name.camelize
   end
+
+  def migration_version
+    if Rails.version.start_with? "5"
```

Comment:
[SUG] Let's do it compatible with higher versions:
```
if Rails.version.to_i >= 5
```

Reply:
This worked perfectly.